emulators/mame/Portfile

```diff
@@ -23,6 +34,68 @@ long_description    The purpose of MAME is to preserve decades of software histo
     the hardware and how it functions. The source code to MAME serves as this \
     documentation.
 
+github.tarball_from archive
+
+#------------------------------------------------------------------------------
+# Patches and Checksums
+#------------------------------------------------------------------------------
+
+# Patch 'language-portuguese_brazil' is only applicable to release 0.226.
+# Fix was submitted, and merged, by Mame maintainers... and will be part of
+# official Mame releases from 0.227 onward. Net-Net, this particular patch can
+# be removed starting with 0.227. Tracked by Mame issue:
+# https://github.com/mamedev/mame/issues/7510
+#
+# Conversely, patch 'scripts-genie-compile-warnings' is not specific to any
+# particular Mame release. Indeed, it is technically not required at all.
+# However, it eliminates spurious warnings due to use of deprecated types
+# and/or calls... and reduces pollution in the build log.
+
+patchfiles          mame-patch-0226-language-portuguese_brazil.diff \
+                    mame-patch-0226-scripts-genie-compile-warnings.diff
+
+checksums           mame-0226.tar.gz \
+                    rmd160  b9e4ae321b7673790d374c63bbe966d1502d6738 \
+                    sha256  7c4c9ec232ba988e65fd29665c9b8e40b5ac3aa9f561eeb107cebbf08ba94baf \
+                    size    196379874
+
+#------------------------------------------------------------------------------
```

Comment:
Please remove all of these extra comment spacing lines:

`#------------------------------------------------------------------------------
`

Reply:
are you strongly against those, Frank? He's put those in on purpose, obviously, as this is a complicated port that has a number of sections. And if he's willing to maintain mame properly, no skin off my nose if this makes it easier for him...

Reply:
Not strongly no. This appears to be the most heavily commented Portfile I've seen in Macports. :) But that really is not a bad thing of course. Anyway, I'm fine with the comments themselves, but the padding seems unnecessary and is not in any other Portfile.

Reply:
I'll defer to the maintainer on this.

Reply:
It's up to you guys. While I find the section separators helpful personally, I'm also a Portfile newbie. And I'd hate to see this be an example of Portfile Don'ts.  :-)

Reply:
Another thought: Since the Mame project is slated to release 0.227 within the next month or two -- and there are also some further refinements I'd like to make to this port -- I'm planning to release a new revision within the next month anyway.

So can a non-functional change like this, wait until then? Thoughts?

Reply:
It's fine to leave them in this PR.